sdk/spring/README.md

```diff
@@ -56,24 +56,24 @@ Below packages are available with latest release version. **We recommend users t
-[spring-cloud-azure-feature-management-web][spring-cloud-azure-feature-management-web-readme] | N/A 
-[spring-cloud-azure-feature-management][spring-cloud-azure-feature-management-readme] | N/A 
-[spring-cloud-starter-azure-appconfiguration-config][spring-cloud-starter-azure-appconfiguration-config-readme] | N/A 
-[azure-spring-cloud-starter-cache][azure-spring-cloud-starter-cache-readme] | [![Maven Central][azure-spring-cloud-starter-cache-mvn-2.x.x-img]][azure-spring-cloud-starter-cache-mvn-2.x.x] 
-[azure-spring-cloud-stream-binder-eventhubs][azure-spring-cloud-stream-binder-eventhubs-readme] | [![Maven Central][azure-spring-cloud-stream-binder-eventhubs-mvn-2.x.x-img]][azure-spring-cloud-stream-binder-eventhubs-2.x.x] 
-[azure-spring-cloud-starter-eventhubs-kafka][azure-spring-cloud-starter-eventhubs-kafka-readme] | [![Maven Central][azure-spring-cloud-starter-eventhubs-kafka-mvn-2.x.x-img]][azure-spring-cloud-starter-eventhubs-kafka-mvn-2.x.x]
-[azure-spring-cloud-starter-eventhubs][azure-spring-cloud-starter-eventhubs-readme] | [![Maven Central][azure-spring-cloud-starter-eventhubs-mvn-2.x.x-img]][azure-spring-cloud-starter-eventhubs-mvn-2.x.x]
-[azure-spring-integration-eventhubs][azure-spring-integration-eventhubs-readme] | [![Maven Central][azure-spring-integration-eventhubs-mvn-2.x.x-img]][azure-spring-integration-eventhubs-mvn-2.x.x]
-[azure-spring-cloud-stream-binder-servicebus-queue][azure-spring-cloud-stream-binder-servicebus-queue-readme] | [![Maven Central][azure-spring-cloud-stream-binder-servicebus-queue-mvn-2.x.x-img]][azure-spring-cloud-stream-binder-servicebus-queue-mvn-2.x.x]
-[azure-spring-cloud-stream-binder-servicebus-topic][azure-spring-cloud-stream-binder-servicebus-topic-readme] | [![Maven Central][azure-spring-cloud-stream-binder-servicebus-topic-mvn-2.x.x-img]][azure-spring-cloud-stream-binder-servicebus-topic-mvn-2.x.x]
-[azure-spring-cloud-starter-servicebus][azure-spring-cloud-starter-servicebus-readme] | [![Maven Central][azure-spring-cloud-starter-servicebus-mvn-2.x.x-img]][azure-spring-cloud-starter-servicebus-mvn-2.x.x]
-[azure-spring-integration-servicebus][azure-spring-integration-servicebus-readme] | [![Maven Central][azure-spring-integration-servicebus-mvn-2.x.x-img]][azure-spring-integration-servicebus-mvn-2.x.x]
-[azure-spring-cloud-starter-storage-queue][azure-spring-cloud-starter-storage-queue-readme] | [![Maven Central][azure-spring-cloud-starter-storage-queue-mvn-2.x.x-img]][azure-spring-cloud-starter-storage-queue-mvn-2.x.x]
-[azure-spring-integration-storage-queue][azure-spring-integration-storage-queue-readme] | [![Maven Central][azure-spring-integration-servicebus-mvn-2.x.x-img]][azure-spring-integration-servicebus-mvn-2.x.x]
+Artifact Id | Version for Spring Boot 2.3.x and 2.2.x | Version for Spring Boot 2.4.x
+------ |--------- |---
+[spring-cloud-azure-appconfiguration-config-web][spring-cloud-azure-appconfiguration-config-web-readme] | N/A | N/A 
```

Comment:
app-configuration is released. it's not `N/A` for both spring-boot 2.2 and 2.3.

Reply:
I removed this section, these items should not be listed here. They belong to another Spring Cloud table, which is for track1 Azure Spring Cloud libs. Will update the table after releasing the app config modules.